### ISSUE TITLE
[BITV] User status dialog a11y issues

### ISSUE THREAD

**Issue by PVince81** [nextcloud/server#33745]
* [x] Needs role="dialog", aria-modal="true" => https://github.com/nextcloud/nextcloud-vue/pull/3113
* [x] #35098
* [x] Close button needs to be moved into the dialog  => already there on master
* [x] Focus and active state needs to look different from each other => https://github.com/nextcloud/server/pull/35170
* [ ] "Clear status after" menu entries need border for focus feedback  => Multiselect issue, see https://github.com/nextcloud/nextcloud-vue/issues/2504
* [x] Main screen can be scrolled when this modal is open => @PVince81 this is already done, no?

**Comment by PVince81** [comment]
@CarlSchwan FYI

**Comment by JuliaKirschenheuter** [comment]
> * Focus and active state needs to look different from each other

I've done it only for OnlineStatusSelect and PredefinedStatus. It makes no sense to change it on CustomMessageInput (input alone has no active state https://html.spec.whatwg.org/multipage/semantics-other.html#selector-active). All other components coming from NC Vue and this issue should be fixed there.

**Comment by JuliaKirschenheuter** [comment]
> "Clear status after" menu entries need border for focus feedback 

Moved to multiselect issue, https://github.com/nextcloud/nextcloud-vue/issues/2504

Closing this issue.
